CMakeLists.txt

```diff
@@ -109,10 +108,7 @@ include(CheckTypeSize)
 check_type_size(ssize_t SSIZE_T)
 check_type_size(size_t SIZEOF_SIZE_T)
 
-configure_file(
-  ${WABT_SOURCE_DIR}/src/config.h.in
-  ${WABT_BINARY_DIR}/config.h
-)
+configure_file(src/config.h.in config.h @ONLY)
```

Comment:
Why is this change?

Reply:
Using only one variable-expansion syntax makes it marginally less likely that something gets accidentally "configured". For example, a comment describing a CMake variable like `// this value is derived from ${PROJECT_VERSION} in CMake`.

Reply:
It's a mix of style and defensiveness that comes from being bitten by such issues over the years. I removed the `${WABT_SOURCE_DIR}` and `${WABT_BINARY_DIR}` expansions because they're redundant and hinder readability.